Atomic/syscontainers.py

```diff
@@ -296,9 +296,9 @@ def _get_remote_location(self, remote_input):
         remote_path = self._resolve_remote_path(remote_input)
         if remote_path:
             remote_rootfs = os.path.sep.join([remote_path, "rootfs"])
-            if os.path.exists(remote_path):
+            if os.path.exists(remote_rootfs):
                 util.write_out("The remote rootfs for this container is set to be {}".format(remote_rootfs))
-            elif os.path.exists(os.path.sep.join([remote_input, "usr"])):  # Assume that the user directly gave the location of the rootfs
+            elif os.path.exists(os.path.sep.join([remote_path, "usr"])):  # Assume that the user directly gave the location of the rootfs
```

Comment:
This is to handle the input with an extra `/` in it. For instance, we can have a input value as `/xx/rootfs/` as remote input, and because we have `os.path.sep.join` here, we will have `/xx/rootfs//usr` in the statement. 

If we change to remote_path(which is real path without the extra /), the above case mentioned will be fine.